### PR TITLE
Templates: Reset page/element selection and `globalStoryStyles`

### LINKED ISSUE #7227: Templates: Reset page/element selection and `globalStoryStyles`
## Task Description

As mentioned at https://github.com/google/web-stories-wp/pull/6988/files#r614312125, some templates do have `current` (the page selection) and `selection` (the selected element(s)) set.

We should reset those in all templates.

`current` should be set to `null`
`selection` should be an empty array

A new test should be added in `packages/templates/src/test/raw.js` to test for this.

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
Template cleanup

## Summary
Remove unneeded properties from templates

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
NA

## Testing Instructions
NA

### QA
NA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT
NA

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7227

### CODE REVIEW

File: packages/templates/src/test/raw.js [hunk 1]
```diff
@@ -83,4 +83,20 @@ describe('raw template files', () => {
       }
     }
   );
+
+  // @see https://github.com/google/web-stories-wp/pull/5889
+  it.each(templates)(
```
Comment (BrittanyIRL) [inline packages/templates/src/test/raw.js:88]: 👍

**Review by BrittanyIRL — APPROVED** [review]
🚀

File: packages/templates/src/test/raw.js [hunk 2]
```diff
@@ -83,4 +83,20 @@ describe('raw template files', () => {
...
+    '%s template should not contain extraneaous properties',
```
Comment (samwhale) [inline packages/templates/src/test/raw.js:89]: this is so dope dude

**Review by samwhale — APPROVED** [review]

File: packages/templates/src/test/raw.js (hunk 2 above)
Comment (maxyinger) [reply packages/templates/src/test/raw.js:89]: Thanks I copy and pasted it 😅

Comment (maxyinger) [reply packages/templates/src/test/raw.js:89]: speaking of which, I need to update the comment above

**Review by swissspidy — APPROVED** [review]
Thank you!

**Pull request merged**